docs/developer/accessibility.md

```diff
@@ -0,0 +1,35 @@
+# Accessibility
+
+FAST-DNA has worked diligently from the ground up to build a component framework that meets all WCAG 2.0 AA standards. Every new component or website revision involves a rigorous approval process to validate and document compliance.
+
+Active collaboration occurs with both Microsoft’s legal, accessibility, and marketing teams to deliver inclusive and accessible experiences.
+
+To learn more about Microsoft's commitment visit their [accessibility](https://www.microsoft.com/en-us/accessibility) website.
+
+Compliments of [@rachelnabors](https://github.com/rachelnabors)
+
+## Browsers
+
+The base component packages support IE10+ and all modern browsers.
+
+### Reporting browser bugs
+
+* [Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/)
+* [Firefox](https://bugzilla.mozilla.org/)
+* [Safari](https://bugs.webkit.org/)
+* [Chrome](https://bugs.chromium.org/p/chromium/issues/list)
+* [Opera](https://bugs.opera.com/wizard/desktop)
+
+## Animation accessibility
+
+* End users should have a mechanism to reduce or remove animations from their experience. The Safari devs have proposed a media query for reduced animation. ("prefers-reduced-motion")
+* Some users with vestibular disorders or other cognitive disabilities have problems with movement. A safe fallback for nearly all users is fade animations.
+* Flashing animations can cause seizures and should be avoided. Good data on the accessibility of animation is scarce.
+
+## Microsoft resources
```

Comment:
Is this here simply for the sake that we are under the MSFT github account?

Reply:
This is here for Microsoft users that want to learn more about Microsoft Accessibility.  But, removing, as it's not key for our project.